fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/SortedQuerySegmentAggregator.java

```diff
@@ -254,7 +256,7 @@ private String buildAggregateExpression(SortParameter sortParm, int sortParmInde
             attributeNames.add(STR_VALUE);
             break;
         case DATE:
-            attributeNames.add(DATE_VALUE);
+            attributeNames.add(DATE_START);
```

Comment:
Is it ok to use DATE_START and no DATE_END?  I'm not sure if I really understand how this getValueAttributeNames is used...

Reply:
Yes, in this case it's for a consistent SORT order.  If you add DATE_END as well, it'll lead to very irregular sorts.